FOR-DEVS.md

```diff
@@ -68,13 +68,20 @@ If you are part of theCourseForum engineering team, follow the instructions belo
     where `db_dump_file.sql` should be replaced with the latest DB file (e.g. `launch.sql`)
 7. Go to http://localhost:8000 in your browser and make sure everything works!
 
+### Windows setup (recommended)
+1. Install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+2. Install Docker
+    - If you're using Windows 10 Pro, Enterprise, or Education, then follow [these instructions](https://docs.docker.com/docker-for-windows/install/) to install Docker Desktop, which comes with Docker and Docker Compose. A free copy of Windows 10 Education is available from UVA.
+    - If you're using Windows 10 Home, which is what comes with most laptops, then follow [these instructions](https://docs.docker.com/docker-for-windows/install-windows-home/). The setup is a bit more complicated because you'll first need to setup [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install-win10#manual-installation-steps). WSL, or Windows Subsystem for Linux, is like a Linux virtual machine that runs in the command line (meaning there's no GUI). It allows Docker to run by using Linux behind the scenes. Check out the [VS Code instructions](https://code.visualstudio.com/docs/remote/wsl) to learn how to develop with WSL (optional).
+3. Proceed at step 2 in **MacOS and Linux setup**
+
 ### Alternative Setup (Windows and MacOS if above failed)
 0. [Install Vagrant](https://www.vagrantup.com/intro/getting-started/install.html)
 1. `git clone https://github.com/thecourseforum/theCourseForum2.git`
 2. Go into the `theCourseForum2/` folder and then run `vagrant up` to start a VM.
 3. After that's booted, run `vagrant ssh` to run commands inside the VM.
 4. `cd /vagrant`
-5. Start at step 3 in normal setup. You may have to prefix all docker commands with `sudo` (e.g. `sudo docker-compose up`).
+5. Start at step 3 in **MacOS and Linux setup**. You may have to prefix all docker commands with `sudo` (e.g. `sudo docker-compose up`). ----------------------
```

Comment:
What is this line?

Reply:
Oh lol that was a dumb way to remind me to change that line
```suggestion
5. Start at step 3 in **MacOS and Linux setup**. You may have to prefix all docker commands with `sudo` (e.g. `sudo docker-compose up`).
```